### PR TITLE
chore: update master to next in CI scripts and docs

### PR DESCRIPTION
**Pull request by domoritz**
<details>
  <summary>📦 Published PR as canary version: <code>5.2.1--canary.8031.0044b51.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install vega-lite@5.2.1--canary.8031.0044b51.0
  # or 
  yarn add vega-lite@5.2.1--canary.8031.0044b51.0
  ```
</details>

### CODE REVIEW

File: .github/workflows/check.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ jobs:
           sudo mv parallel sem /usr/local/bin
 
       - name: Format
-        if: github.ref != 'refs/heads/master'
+        if: github.ref != 'refs/heads/next'
```
Comment (domoritz) [inline .github/workflows/check.yml:38]: @hydrosquall do we still need this?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ name: Test
 on:
   push:
     branches:
-      - master
+      - next
```
Comment (domoritz) [inline .github/workflows/test.yml:6]: Was this not updated for a reason?

Comment (hydrosquall) [reply .github/workflows/test.yml:6]: It was an oversight, good to include it now

File: .github/workflows/check.yml (hunk 1 above)
Comment (hydrosquall) [reply .github/workflows/check.yml:38]: I think so, assuming the goal here is to warn someone if they tried to open a PR with some unformatted code

**Review by hydrosquall — APPROVED** [review]
Looks good, thanks. After this goes in, I'm think might consider deleting the `master` branch altogether (after merging the commits it absorbed accidentally to next first)

**Comment by domoritz** [comment]
Go for it.

**Comment by hydrosquall** [comment]
Unfortunately, there seems to be a bug with Github, I've queried for all PRs that may have master as the base (TIL via [Stackoverflow](https://stackoverflow.com/questions/20547198/how-can-i-filter-all-github-pull-requests-for-a-specific-target-branch)), and the list is empty

- https://github.com/vega/vega-lite/pulls?q=is%3Aopen+is%3Apr+base%3Amaster

However, the "delete" button isn't working in my UI.

Image 2022-03-07 at 8 18 12 PM

Hopefully this is a caching issue that will go away tomorrow.

**Comment by domoritz** [comment]
Hmm, I'm still getting this error today.

**Comment by hydrosquall** [comment]
@domoritz me too...

One option might be to rename the branch to something like `master-do-not-use-see-pull-8031`, since the "rename" function appears to still be working. I'm going to do that as a starting point, since hard to find is almost as good as deleted.

**Comment by hydrosquall** [comment]
Image 2022-03-09 at 9 13 43 PM

Success! (the `master` name persisted in the delete action, despite my going through the rename step first).

**Comment by domoritz** [comment]
Lol and 🎉

**Pull request merged**